src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -847,7 +847,7 @@ public PurchaseResult execute(final String merchantAccount, final PaymentData pa
                                                      } if(authoriseKlarnaPayment) {
                                                          Map<String, String> responseData = AdyenDao.fromAdditionalData(existingAuth.getAdditionalData());
                                                          paymentData.getPaymentInfo().setAuthResponseData(responseData);
-                                                         return adyenPort.completeKlarnaPaymentAuth(merchantAccount, paymentData, userData);
+                                                         return adyenPort.authoriseKlarnaPayment(true, merchantAccount, paymentData, userData);
```

Comment:
I think `completeKlarnaPaymentAuth` is more readable than  `authoriseKlarnaPayment` with `boolean` param. You can always abstract the core implementation, and this core implementation of Auth can be called by both the functions based what they are suppose to do.